src/tools/singlejar/BUILD

```diff
@@ -100,7 +99,7 @@ cc_test(
 cc_test(
     name = "desugar_checking_test",
     srcs = [
-        "combiners_test.cc",
+        "desugar_checking_test.cc",
```

Comment:
Hm, was `desugar_checking_test` never actually executing?

Reply:
I think it never actually executed. See the commit that introduced `desugar_checking` library: https://github.com/bazelbuild/bazel/commit/5f911199c250e77a832be2e4f69017c69d446273